### PR TITLE
feat(ml):added a python file

### PR DESCRIPTION
**Pull request by Rusali28**

### CODE REVIEW

**Comment by saxenabhishek** [comment]
Great Work 💯

File: testnew.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("Hello World")
```
Comment (saxenabhishek) [inline testnew.py:1]: Can you change this to say "Hey world"

**Review by saxenabhishek — CHANGES_REQUESTED** [review]
Just change that line

File: testnew.py (hunk 1 above)
Comment (Rusali28) [reply testnew.py]: Yes I did it

**Review by saxenabhishek — APPROVED** [review]
Nice work

**Pull request merged**